src/dataServices/dynamoDbParamBuilder.ts

```diff
@@ -60,9 +60,14 @@ export default class DynamoDbParamBuilder {
         return params;
     }
 
-    static buildGetResourcesQueryParam(id: string, maxNumberOfVersions: number, projectionExpression?: string) {
+    static buildGetResourcesQueryParam(
+        id: string,
+        maxNumberOfVersions: number,
+        tenantId: string,
+        projectionExpression?: string,
+    ) {
         const params: any = {
-            TableName: RESOURCE_TABLE,
+            TableName: RESOURCE_TABLE + tenantId,

```

Comment:
Change tenetid to not have the `-`
```suggestion
            TableName: `${RESOURCE_TABLE}-${tenantId}`,

```

Reply:
Fixed as part of commit 691de3a679f1a98b3ff29e422139bf263f75ac17